README.md

```diff
@@ -76,6 +136,7 @@ Allow user to browse and save favorite destinations and more while providing wea
 ---
 
```

Comment:
Maybe put my name back on it?

Reply:
your name is there.

Reply:
Make sure you expand the file. When there are no changes, github collapses those lines.

Reply:
Ok, sorry. I'm looking at everything on mobile so it's a little different.